pkgs/tools/misc/starship/default.nix

```diff
@@ -33,7 +33,7 @@ rustPlatform.buildRustPackage rec {
     export HOME=$TMPDIR
   '';
 
-  postInstall = ''
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Comment:
the presets below are not conditioned on `canExecute`

Reply:
Nice catch. Fixed.

Also rebased on master, and additionally had to fix the formatting of maa-cli now to get a green CI…